docs/sequence_diagrams/notification-schedulerFlow.plantuml

```diff
@@ -0,0 +1,57 @@
+@startuml
+' declate title
+title 10.6. Notification Scheduler Flow
+autonumber
+
+' declare actors
+'collections "Notification-Topic" as TOPIC_NOTIFY
+'boundary "Central Event Processor" as CEP
+control "Action Agent" as AA
+database "CEP Store" as DB
+
+box "Central Services" #LightYellow
+'participant TOPIC_NOTIFY
+end box
+box "Central Event Processing Service" #LightGreen
+'participant CEP
+participant AA
+participant DB
+end box
+
+' start flow
+Group Scheduler Flow
+    activate AA
+    AA -> AA
+    note right of AA #lightGrey
+    Scheduled time for certain
+    event **<actionId>** has been reached
```

Comment:
what's the meaning of < >? it's already emphasised with **bold**

Reply:
Removed the <>.

Reply:
Removed

Event **actionId**